cluster_health_check.md

```diff
@@ -0,0 +1,87 @@
+Etcd Cluster Health Check
+========
+
+Synopsis
+========
+
+```nginx
+http {
+    # required declares a shared memory zone to store endpoints's health status
+    lua_shared_dict healthcheck_shm 1m;
+
+    server {
+        location = /healthcheck {
+            content_by_lua_block {
+                local etcd, err = require "resty.etcd" .new({
+                    protocol = "v3",
+                    http_host = {
+                        "http://127.0.0.1:12379", 
+                        "http://127.0.0.1:22379",
+                        "http://127.0.0.1:32379",
+                    },
+                    user = 'root',
+                    password = 'abc123',
+                    # minimal configuration to enable etcd cluster health check
+                    cluster_healthcheck ={
```

Comment:
please confirm, the `cluster_healthcheck ` should be optional for user

Reply:
yes, it 's optional for user. update annotation